tutorials/cp-starter-digitalexp-mobilecards-2-create/cp-starter-digitalexp-mobilecards-2-create.md

```diff
@@ -0,0 +1,210 @@
+
+[ACCORDION-BEGIN [Step 1: ](Create a Timesheet card)]
+
+Enter Name as `TimeSheetCard` and click **Save**.
+
+![MobileCards](img_1.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Create To Do card)]
+
+Choose **Create a New Card**.
+
+![MobileCards](img_4.png)
+
+Provide the following values for the new card:
+
+| Field | Value |
+|----|----|
+| **Name** | `ToDosCard` |
+| **Destination** | `SFDEMO_APIHUB` |
+| **Template Source** | `Template Manager` |
+| **HTML Template** | `SuccessFactors My To Dos` |
+
+> If you see a pop-up, choose **OK**.
+
+![MobileCards](img_5.png)
+
+Choose **Save**.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 3: ](Set Card status to Productive)]
+
+>By default, status of newly created cards is **Development**. Cards can be subscribed to only when they are in a **Productive** state.
+
+To change the card status, click on `TimeSheetCard`.
+
+![MobileCards](img_5.1.png)
+
+Under **Actions**, click on highlighted icon to change the state to **Productive**.
+
+![MobileCards](img_5.2.png)
+
+Choose **Yes** to confirm.
+
+![MobileCards](img_5.3.png)
+
+You will notice that the **State** has been changed to **Productive**.
+
+![MobileCards](img_5.4.png)
+
+Repeat the above step for the `ToDosCard`. Status of both the cards should be **Productive**.
+
+![MobileCards](img_5.5.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Configure SAP Mobile Cards client)]
+
+>Ensure that you are choosing the right device platform **above**.
+
+>Make sure you have installed the SAP Mobile Cards app on your phone.
+
+You can now start on-boarding your SAP Mobile Cards client onto SAP Cloud Platform.
+
+Choose the **APIs** tab to view the registration QR code. Scan the QR code applicable to your device type (Android or iOS).
+
+![MobileCards](img_101.png)
+
+[OPTION BEGIN [Android]]
+
+Tap **SCAN QR CODE** to begin the on-boarding process.
+
+![MobileCards](img_1.jpg)
+
+Tap **I AGREE** on `End User License Agreement`.
+
+![MobileCards](img_2.jpg)
+
+Enter your SAP Cloud Platform credentials, and choose **Log On**.
+
+![MobileCards](img_3.jpg)
+
+Choose a passcode with at least 5 characters to unlock the app, and tap **NEXT**.
+
+![MobileCards](img_4.jpg)
+
+Confirm the passcode and tap **DONE**.
+
+![MobileCards](img_5.jpg)
+
+Optionally, enable fingerprint.
+
+![MobileCards](img_6.jpg)
+
+Tap the **+** icon to add subscriptions.
+
+![MobileCards](img_7.jpg)
+
+Tap **Subscriptions**.
+
+![MobileCards](img_8.jpg)
+
+Tap **SUBSCRIBE** for `TimeSheetCard`.
+
+![MobileCards](img_9.jpg)
+
+Tap on arrow key to navigate back to **All Subscriptions** list.
+
+![MobileCards](img_201.png)
+
+Tap `ToDosCard` to subscribe it.
+
+![MobileCards](img_202.png)
+
+Tap **SUBSCRIBE**.
+
+![MobileCards](img_10.jpg)
+
+You can now view both the cards.
+
+![MobileCards](img_11.jpg)
+
+[OPTION END]
+
+[OPTION BEGIN [iOS]]
+
+Scan the QR code with your camera and select the toast message to launch the SAP Mobile Cards app.
+
```

Comment:
QR-code is incorrect, 
for iOS not working tool, 
for Android opens API L**** Apotheke

Reply:
Are you pointing to QR code from tutorial? If so, i have blurred it. 
You should be able to scan the QR code from your account cockpit.

Reply:
> Are you pointing to QR code from tutorial? If so, i have blurred it.
> You should be able to scan the QR code from your account cockpit.

no, I scanned from the cockpit

Reply:
Did you already install Mobile Cards client on your device? Are you trying on iOS or Android?